srcpkgs/python-incremental/template

```diff
@@ -1,26 +1,25 @@
 # Template file for 'python-incremental'
 pkgname=python-incremental
-version=17.5.0
-revision=6
+version=21.3.0
+revision=1
```

Comment:
So, twisted stopped working with py2 but incremental is fine?

Reply:
Yes, apparently. Just tested it with python2 to confirm.